tutor/src/components/notes.js

```diff
@@ -419,12 +423,33 @@ class NotesWidget extends React.Component {
         />
 
         <div className="annotater-content" ref={this.setElement}>
-          <Overlay
-            id="notes-summary"
-            visible={NotesUX.isSummaryVisible}
+          <Modal
+            show={NotesUX.isSummaryVisible}
             onHide={NotesUX.hideSummary}
-            renderer={this.renderSummaryPage}
-          />
+            onShow={this.setupModalScrollTop}
+            dialogClassName="notes-modal"
+            scrollable={true}
+          >
+            <Modal.Header
+              closeButton={true}
+              closeLabel={'Close'}
+            >
+              <Modal.Title>My Highlights and Notes</Modal.Title>
+            </Modal.Header>
+            <Modal.Body>
+              <SummaryPage
+                notes={this.props.course.notes}
+                onDelete={this.onNoteDelete}
+                page={this.props.page}
+              />
+              <a href="#"
+                className="modal-scroll-top"
+                onClick={this.onModalScollTop}
+              >
+                <Icon type="angle-up" color="#fff" />
+              </a>
+            </Modal.Body>
```

Comment:
Icon has a click handler, I think this could be:

`<Icon type="angle-up" color="#fff" onClick={this.onModalScollTop} className="modal-scroll-top" />`

Reply:
Good to know for the future, I see it can be passed class names too. It does come with some unwanted style baggage because of the other classes that are automatically added; might be a nice future refactor to allow passing an element.

Reply:
Thanks for mentioning this, I circled back and realized `buttonProps` was passing to the bootstrap button, so I was able to override exactly what I wanted 👍